R/dsl.R

```diff
@@ -103,3 +103,48 @@ generate_random_id <- function(prefix, id_length = 20) {
   random_id <- paste(sample(letters, id_length, replace = TRUE), collapse = "")
   paste0(prefix, "-", random_id)
 }
+
+#' Create Semantic UI Message
+#'
+#' This creates a message using Semantic UI
+#'
+#' @param header Header of the message
+#' @param content Content of the message. If it is a vector, creates a list of vector's elements
+#' @param type Type of the message. Look at https://semantic-ui.com/collections/message.html for all possibilities.
+#' @param icon If the message is of the type 'icon', specify the icon. Look at http://semantic-ui.com/elements/icon.html for all possibilities.
+#' @param closable Determines whether the message should be closable. Default is FALSE - not closable
+#'
+#' @export
+
+closable_messages <- "$('.message .close')
```

Comment:
This should be above the documentation. Make it UPPER_CASE to indicate it's a constant value.